test/Microsoft.Framework.Configuration.FileExtensions.Test/FileConfigurationBuilderExtensionsTest.cs

```diff
@@ -55,7 +55,7 @@ public void GetBasePath_ReturnEmptyIfNotSet()
             var actualPath = configurationBuilder.GetBasePath();
 
             // Assert
-            Assert.Equal(string.Empty, actualPath);
+            Assert.Equal(Directory.GetCurrentDirectory(), actualPath);
```

Comment:
`BasePath` and `CurrentDirectory` can be different. Are you sure that is never the case when we run our tests from the command line or on CI?